tests/unit/services/experiments-test.js

```diff
@@ -0,0 +1,73 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | experiments', function (hooks) {
+  let experiments
+  let storageMock
+  let activityExpectationsMock
+
+  const solvedChallenge = {id: "13"}
+  setupTest(hooks);
+
+  hooks.beforeEach(function () {
+    experiments = this.owner.lookup('service:experiments')
+    experiments.set('group', "treatment")
+    experiments.set('decompositionTreatmentLength', 2)
+
+    storageMock = {
+      solvedChallenges: [],
+      getSolvedChallenges(){
+        return this.solvedChallenges
+      }
+    }
+    experiments.storage = storageMock
+
+    activityExpectationsMock = {
+      _decomposition: true,
+
+      combinedExpectations(){
+        return {decomposition: this._decomposition}
+      },
+      expectationsExist() { return true },
+      mergedExpectations() {
+        return this.combinedExpectations()
+      }
+    }
+
+    experiments.activityExpectations = activityExpectationsMock
+  })
+
+  test('Should show congratulations modal when group is not affected', function (assert) {
+    experiments.set('group', "notAffected")
+    assert.ok(experiments.shouldShowCongratulationsModal())
+  })
+
+  test('Should NOT show congratulations modal when group is affected and feedback is enabled', function (assert) {
+    assert.notOk(experiments.shouldShowCongratulationsModal())
+  })
```

Comment:
Creo que agregaría un test del primero falso y segundo verdadero, tipo: `Should show congratulations modal when group is affected and feedback is disabled`.

Reply:
De acuerdo. Y ojo con lo que mencioné sobre que es el ControlModal.

Reply:
... que por cierto ahora que veo cómo lo estamos usando debería llamarse algo así como "SimpleGoalListModal" o algo así, que dé idea de que están las expects pero que no se muestra si se cumplieron o no. O quizás "NonScoredExpectationModal" en lugar de "Control Modal" y "ScoredExpectationModal" en lugar de "TreatmentModal" o algo así. Así lo separamos de la idea de "Control" o "Treatment" que siempre me molestó que estuviera nombrado así. Una cosa es el estilo de modal que muestro y otra si está asociado a qué grupo experimental o qué fase del experimento.